README.adoc

```diff
@@ -1 +1,82 @@
 = Shell Function Invoker image:https://ci.projectriff.io/api/v1/teams/main/pipelines/riff/jobs/build-shell-function-invoker-container/badge[Shell Invoker Status, link=https://ci.projectriff.io/teams/main/pipelines/riff/build-shell-function-invoker-container/builds/latest]
+
+== Purpose
```

Comment:
This should explain that it's gonna server as a base layer in a docker container

Reply:
Explained.